Sources/ComposableArchitecture/Effects/Throttle.swift

```diff
@@ -0,0 +1,110 @@
+import Combine
+import Dispatch
+import Foundation
+
+extension Effect {
+  /// Throttles an effect so that it only publishes one output per given interval.
+  ///
+  /// The throttling of an effect is with respect to actions being sent into the store. So, if
+  /// you return a throttled effect from an action that is sent with high frequency, the effect
+  /// will be executed at most once per interval specified.
+  ///
+  /// > Note: It is usually better to perform throttling logic in the _view_ in order to limit
+  /// the number of actions sent into the system. Only use this operator if your reducer needs to
+  /// layer on specialized logic for throttling. See <docs:Performance> for more information of why
```

Comment:
this `<docs:Performance>` doesn't link correctly. is it not allowed to link to articles from docs of a method?

Reply:
It might need to be `<docs:/Dependencies/Performance>` to work, but cross-module linking is only supported in the Xcode 15 beta.

Reply:
Oh wait, it's `<doc:>` not `<docs:>`. Fixed!